packages/elasticsearch-plugin/src/elasticsearch.service.ts

```diff
@@ -445,29 +497,30 @@ export class ElasticsearchService implements OnModuleInit, OnModuleDestroy {
             productName: source.productName,
             productVariantId: source.productVariantId.toString(),
             productVariantName: source.productVariantName,
-            facetIds: source.facetIds as string[],
-            facetValueIds: source.facetValueIds as string[],
-            collectionIds: source.collectionIds as string[],
+            facetIds: source.productFacetIds as string[],
+            facetValueIds: source.productFacetValueIds as string[],
+            collectionIds: source.productCollectionIds as string[],
             sku: source.sku,
             slug: source.slug,
             price: {
-                min: source.priceMin,
-                max: source.priceMax,
+                min: source.productPriceMin,
+                max: source.productPriceMax,
             },
             priceWithTax: {
-                min: source.priceWithTaxMin,
-                max: source.priceWithTaxMax,
+                min: source.productPriceWithTaxMin,
+                max: source.productPriceWithTaxMax,
             },
             channelIds: [],
             score: hit._score || 0,
         };
-        this.addCustomMappings(result, source, this.options.customProductMappings);
+        ElasticsearchService.addCustomMappings(result, source, this.options.customProductMappings, true);
         return result;
     }
 
-    private getSearchResultAssets(
-        source: ProductIndexItem | VariantIndexItem,
-    ): { productAsset: SearchResultAsset | undefined; productVariantAsset: SearchResultAsset | undefined } {
+    private static getSearchResultAssets(source: ProductIndexItem | VariantIndexItem): {
```

Comment:
Why `static`?

Reply:
It was WebStorm suggestion. In Python static functions are rather quicker. Revert it